pennylane_lightning/core/src/bindings/BindingsBase.hpp

```diff
@@ -21,6 +21,7 @@
 
 #include <pybind11/pybind11.h>
 
+#include "Constant.hpp"
```

Comment:
What's in use from Constant vs ConstantUtil?

Reply:
yea, just as @vincentmr said! Thanks